src/components/question-grid/QuestionGrid.tsx

```
@@ -93,6 +95,15 @@ export function QuestionGrid({
                 </div>
 
                 <div className="flex items-center gap-1 opacity-0 group-hover:opacity-100 transition-opacity shrink-0">
+                  {question.status === 'private' && onMakePublic && (
+                    <button
+                      onClick={(e) => { e.stopPropagation(); onMakePublic(question._id); }}
+                      className="p-1.5 rounded-full hover:bg-gray-100 dark:hover:bg-gray-800 transition-colors text-gray-400 hover:text-blue-500"
+                      title="Make Public (Requires Review)"
+                    >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)